README.rst

```diff
@@ -18,17 +18,20 @@ Examples
 
    import json
 
+   from datetime import datetime, timedelta
+
    from jwt import (
        JWT,
        jwk_from_dict,
        jwk_from_pem,
    )
+   from jwt.utils import get_int_from_datetime
 
    message = {
        'iss': 'https://example.com/',
        'sub': 'yosida95',
-       'iat': 1485969205,
-       'exp': 1485972805,
+       'iat': get_int_from_datetime(datetime.utcnow()),
+       'exp': get_int_from_datetime(datetime.utcnow() + timedelta(hours=1)),
```

Comment:
Naive- `datetime` always handled as it is a local time, so calling `datetime.timestamp` on a naive UTC `datetime` causes the unnecessary time subtraction of the offset from UTC. So we should call `get_int_from_datetime` with `datetime.now()` for naive-datetime or `datetime.now(timezone.utc)` for aware-datetime.